src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        int colorIndex = new Random().nextInt(Color.values().length - 1); // -1 bound exception
+        return Color.values()[colorIndex].name(); // return random index Color
```

Comment:
lets make Random as class level variable, not to create new instance of Random class every time we call getRandomColor() method. And lets remove unnecessary comments